cmd/minishift/cmd/config/config.go

```diff
@@ -77,6 +77,9 @@ var (
 	OpenshiftEnv      = createFlag("openshift-env", nil, nil, nil, false)
 	Metrics           = createFlag("metrics", SetBool, nil, nil, true)
 	Logging           = createFlag("logging", SetBool, nil, nil, true)
+	// future enabled flags
+	ServiceCatalog      = createFlag("service-catalog", SetBool, nil, nil, true)
+	ExtraClusterUpFlags = createFlag("extra-clusterup-flags", SetString, nil, nil, true)
```

Comment:
@gbraad I am not sure why you have two flags i.e. if you going to accept `cluster up` flags through `extra-clusterup-flags` then we do not need `--service-catalog` right?